# CV Jens Kleinjung
```
Dr. Jens Kleinjung
Computational Biology and Chemistry
The Francis Crick Institute
jens.kleinjung@crick.ac.uk
07707467523
```

## Research Interests & Skills
Starting as an experimental protein/peptide chemist, my research activity
has encompassed Biophysics of proteins (NMR and optical spectroscopy) and
Bioinformatics, where I have worked on a wide range of theoretical aspects
of protein sequences and structures.
Among those are the parallelisation of multiple sequence alignment, hybrid
Molecular Dynamics and Monte Carlo simulations, parametrisation of
implicit solvent forces from large-scale simulations, structural alphabets
to describe protein structures in terms of canonical fragments and
applications to detect allosteric signal transmission in proteins.

In recent years I have focused on the analysis of NGS data, in particular
the analysis of single-cell RNA-Seq data. To that end, I have developed an
automated pipeline for the complete processing of 'fastq' input data to the
final table of transcript counts. My current focus is on modelling the
pseudo-temporal gene expression of stem cells along their differentiation
to specific cell lineages.

Beside my research activities, I was responsible for the computational
ecosystem of the Division of Mathematical Biology at the National Institute
for Medical Research (before it was absorbed in the Francis Crick Institute).
The core facility of that Division was a Beowulf cluster with a capacity
of about 1500 cores and 20TB storage. Additionally, I maintained the shared
software, databases and intranet and supported users in software development
and cluster usage.

I am an experienced programmer in C, R and Perl and I have used Linux
for over a decade in system administration and computational research.

## Expertise
* transcriptomics data analysis
* protein sequence and structure analysis
* discrete state models
* molecular simulations
* allostery
* molecular interaction networks
* Linux system administration

## Professional Vita
* since 04.2015
: Bioinformatics Officer, The Francis Crick Institute, London, U.K.
* 10.2005-03.2015
: Senior Investigator Scientist, Division of Mathematical Biology, National Institute for Medical Research, London, U.K.
* 12.2002-09.2005
: Assistant Professor of Bioinformatics, Vrije Universiteit, Amsterdam, The Netherlands
* 01.1998-11.2002
: Postdoctoral Research Fellow, National Institute for Medical Research, London, U.K.
* 12.1996-11.1997
: Postdoctoral Research Fellow, LCPM-CNRS, Nancy, France
* 04.1996-11.1996
: Research assistant, RWTH Aachen University, Germany
* 10.04.1996
: Dr. rer. nat. (PhD Chemistry), RWTH Aachen University, Germany
* 01.10.1992-01.04.1996
: PhD thesis (Chemistry) 'Insulin-receptor interactions', RWTH Aachen University, Germany
* 01.01.1992-30.06.1992
: Diploma thesis (Chemistry) 'Phosphorylated Insulins', RWTH Aachen University, Germany
* 08.1985-12.1991
: Studies in chemistry and biology, RWTH Aachen University, Germany

## Awards & Grants
* 2010/2012
: EMBO long-term fellowship (ALTF 651-2009) to host Dr. J. Baussand
* 2008/2009
: European Science Foundation grant of the programme "Frontiers in Functional Genomics" to host Dr. J. Baussand
* 2008-2010
: EU project ProtFlexPred (Proposal N° 220256) of the programme "Marie-Curie Intra-European Fellowships for Career development" (Call identifier: PEOPLE-2007-2-1-IEF) to host Dr. A. Pandini
* 2007/2008
: European Science Foundation grant of the programme "Frontiers in Functional Genomics" to host Dr. A. Pandini
* 2004
: JSPS Invitation Fellowship (6 weeks) for collaborative research at the Global Scientific Information and Computing Center (GSIC), Tokyo with Dr. M. Ota; GRID application of fast protein folding simulations
* 1995-1997
: Research grant (2 years) 'Synthesis of Phosphorylated Insulins' from German Research Ministry (DFG)
* 1992-1995
: PhD stipend (4 years) of the Land Nordrhine Westphalia for outstanding students

## Talks & Lectures
* 11.03.2014
: Crick Symposium on Quantitative Biology; 'Signal Transduction in Protein Structures'
* 20.11.2013
: BOKU University Wien, Austria; 'Signal Transduction in Protein Structures'
* 15.10.2012
: Radboud University Nijmegen, The Netherlands; 'Functional Analysis of Proteins via Structural Alphabets'
* 05-06.2011
: Guest professorship at the ETH Zürich, Switzerland; Group of Prof. W.F. van Gunsteren (computer-aided chemistry)
* 05.2010
: German Research School for Simulation Sciences, Forschungszentrum Jülich, Germany; Course 'Introduction to Bioinformatics', Group of Prof. P. Carloni, 10 lectures and 3 tutorials
* 05.2009
: SISSA International School for Advanced Studies, Trieste, Italy; Course 'Introduction to Bioinformatics', Group of Prof. P. Carloni, 10 lectures and 3 tutorials
* 03.2009
: University of Liverpool, Liverpool, U.K.; 'Algorithmic Approaches for Protein Structure Analysis and Design'
* 11.2008
: King's College, London, U.K.; 'Alignment Methods for Protein Modelling and Design'
* 09.2008
: University of Cagliari, Cagliari, Italy; 'Principles of Biological Sequence Alignment'
* 06.2005
: Universitá degli Studi di Milano, Milano, Italy; 'Unified Sequence and Structure Alignment'
* 02.2005
: Zentrum für Bioinformatik, Hamburg, Germany; 'Unified Sequence and Structure Alignment'
* 02.2005
: CEINGE Institute, Napoli, Italy; 'Flexible Sequence and Structure Alignment Using Constraints'
* 01.2005
: Groningen Bioinformatics Centre, Groningen, Netherlands; 'Flexible Sequence and Structure Alignment Using Constraints'
* 08.2004
: Quantum Bioinformatics Group, Nara, Japan; 'New Algorithms for Biological Sequence Analysis'
* 08.2004
: National Center of Genetics, Mishima, Japan; 'New Algorithms for Biological Sequence Analysis'
* 07.2004
: Yokohama City University, Yokohama, Japan; 'Recent Progress in Protein Molecular Dynamics Simulations'
* 11.2003
: Biotecnologia 2003, Havana, Cuba; 'Contact-based Sequence Alignment'
* 09.2001
: Biophysical Chemistry 2001, Royal Society of Chemistry, London, U.K.; 'Unfolding Simulations Using Leap-dynamics'
* 02.2001
: BBSRC Bioinformatics Grantholders' Workshop, Hinxton, U.K.; 'Quest: Iterative Sequence Searching'

## Other Professional Activities
* Member of the International Society for Computational Biology
* Academic editor of Scientific Reports
* Academic editor of PLoS ONE
* Reviewer for various research funding organisations

## Programs & Servers
* [POPS](https://mathbio.crick.ac.uk/wiki/POPS)
* [POPSCOMP](https://mathbio.crick.ac.uk/wiki/POPSCOMP)
* [more on GitHub](https://jkleinj.github.io/)

## Publications
* [on ORCID](https://orcid.org/0000-0002-7875-5724)
* [on Google Scholar](https://scholar.google.co.uk/citations?user=ZIZ075AAAAAJ&hl=en)

1. Lasrado  R,  Boesmans  W,  Kleinjung  J,  Pin  C,  Bell  D,  Bhaw  L,  McCallum  S,  Zong
     H,  Luo  L,  Clevers  H,  Berghe  PV  and  Pachnis  V.   **Lineage-dependent  spatial  and
     functional organization of the mammalian enteric nervous system.** *Science* 356(6339)
     722-726 (2017).

2. Gouti M, Delile J, Stamataki D, Wymeersch FJ, Huang Y, Kleinjung J, Wilson V and
     Briscoe J. **A Gene Regulatory Network Balances Neural and Mesoderm Specification
     during Vertebrate Trunk Development.** *Developmental Cell*  41(3) 243-261.e7 (2017).

3. Pandini A, Kleinjung J, Rasool S and Khan S.  **Coevolved Mutations Reveal Distinct
     Architectures for Two Core Proteins in the Bacterial Flagellar Motor.**  *PloS One*  10
     e0142407 (2015).

4. Pandini A, Kleinjung J, Taylor W and Khan S.  **The Phylogenetic Signature Underlying ATP
     Synthase c-ring Dynamics.**  *Biophys  J*  109 975-987 (2015).

5. Gouti M, Tsakiridis A, Wymeersch FJ, Huang Y, Kleinjung J, Wilson V and Briscoe J.
     **In  Vitro  Generation  of  Neuromesodermal  Progenitors  Reveals  Distinct  Roles  for
     Wnt Signalling in the Specification of Spinal Cord and Paraxial Mesoderm Identity.**
     *PLOS  Biology*  12 e1001937 (2014).

6. Kleinjung J and Fraternali F.  **Design and Application of Implicit Solvent Models in
     Biomolecular Simulations.** *Current Opinion in Structural Biology* 25 126-134 (2014).

7. Pernigo S, Fukuzawa A, Pandini A, Holt M, Kleinjung J, Gautel M and Steiner RA.
     **The Crystal Structure of the Human Titin:  Obscurin Complex Reveals a Conserved
     yet Specific Muscle M-Band Zipper Module.**  *J  Mol  Biol*  427 707-714 (2014).

8. Baussand  J  and  Kleinjung  J.  **Specific  Conformational  States  of  Ras  GTPase  upon
     Effector Binding.** *Journal of Chemical Theory and Computation* 9(1) 738-749 (2013).

9. Chakroun N, Fornili A, Prigent S, Kleinjung J, Dreiss CA, Rezaei H and Fraternali F.
      **Decrypting  Prion  Protein  Conversion  into  a  fi-Rich  Conformer  by  Molecular
      Dynamics.**   *Journal  of  Chemical  Theory  and  Computation*  9(5)  2455-2465  (2013).

10. Pandini A, Fornili A, Fraternali F and Kleinjung J.  **GSATools:  analysis of allosteric
      communication  and  functional  local  motions  in  GROMACS  using  a  Structural  Alphabet.**
      *Bioinformatics*  29 2053-2055 (2013).

11. Kleinjung  J  and  Fraternali  F.   **Urea-water  solvation  forces  on  prion  structures.**
      *Journal  of  Chemical  Theory  and  Computation* 8 3977-3984 (2012).

12. Kleinjung J, Scott WRP, Allison JR, van Gunsteren WF and Fraternali F.  **Implicit
      Solvation  Parameters  Derived  from  Explicit  Water  Forces  in  Large-Scale  Molecular
      Dynamics  Simulations.**   *Journal  of  Chemical  Theory  and  Computation*  8(7)  2391-2403 (2012).

13. Pandini  A,  Fornili  A,  Fraternali  F  and  Kleinjung  J.   **Detection  of  allosteric  signal
      transmission by information-theoretic analysis of protein dynamics.**  *FASEB Journal*
      26 868-881 (2012).

14. Kemena C, Taly JF, Kleinjung J and Notredame C.  **STRIKE: Evaluation of Protein
      Multiple  Sequence  Alignments  using  a  Single  Three  Dimensional  Structure.**
      *Bioinformatics*  27 3385-3391 (2011).

15. Fernandes  LP,  Annibale  A,  Kleinjung  J,  Coolen  ACC  and  Fraternali  F.   **Protein
      networks reveal detection bias and species consistency when analysed by information-
      theoretic methods.**  *PLoS  ONE*  5 e12083 (2010).

16. Pandini A, Fornili A and Kleinjung J.  **Structural alphabets derived from attractors
      in conformational space.**  *BMC  Bioinformatics*  11 97 (2010).

17. Schmitz  S,  Schaap  IAT,  Kleinjung  J,  Harder  S,  Grainger  M,  Calder  L,  Rosenthal
      PB,  Holder  AA  and  Veigel  C.   **Malaria  parasite  actin  polymerisation  and  filament
      structure.**  *J.  Biol.  Chem.* 285 36577-36585 (2010).

18. Annibale  A,  Coolen  ACC,  Fernandes  LP,  Fraternali  F  and  Kleinjung  J.   **Tailored
      graph  ensembles  as  proxies  or  null  models  for  real  networks  I:  tools  for  quantifying
      structure.** *Journal  of  Physics  A:  Math.  Theor.* 42 485001 (2009).

19. Autore  F,  Melchiorre  S,  Kleinjung  J,  Morgan  WD  and  Fraternali  F.  **Interaction  of
      Malaria Parasite-Inibitory Antibodies with the Merozite Surface Protein MSP1$_{19}$
      by Computational Docking.**  *Proteins*  66 513-527 (2007).

20. Pandini A, Bonati L, Fraternali F and Kleinjung J.  **MinSet:  A general approach to
      derive maximally representative database subsets by using fragment dictionaries and
      its application to the SCOP database.**  *Bioinformatics*  23 515-516 (2007).

21. Kleinjung  J  and  Fraternali  F.  **POPSCOMP:  An  automated  interaction  analysis  of
      biomolecular complexes.**  *Nuc.  Acids  Res.* 33 W342-W346 (2005).

22. Simossis  V,  Kleinjung  J  and  Heringa  J.   **Homology-extended  sequence  alignment.**
      *Nuc.  Acids  Res.* 33 816-824 (2005).

23. Fraternali  F  and  Kleinjung  J.   **Molecular  simulations  in  structural  genomics.**   In
      *Encyclopedia  of  Genetics,  Genomics,  Proteomics  and  Bioinformatics*,  volume  4.6,
      (74). John Wiley & Sons (2004).

24. Shojaee-Moradie F, Chan M P, Telfer M A, Brandenburg D, Sundermann E,
   	  Eckey H, Kleinjung J, Sch{\"u}ttler A and Jones R H.  **Effect  of
      thyroid hormone binding proteins on insulin receptor binding of B1-thyronine-insulin
      analogues.** *Biochem. J.* 381 51-57  (2004).

25. Kleinjung  J,  Romein  J,  Lin  K  and  Heringa  J.   **Contact-based  sequence  alignment.**
      *Nuc.  Acids  Res.* 32 2464-2473 (2004).

26. Vaccaro L, Cross KJ, Kleinjung J, Straus SK, Thomas DJ, Wharton SA, Skehel JJ
      and  Fraternali  F.  **Plasticity  of  Influenza  Haemagglutinin  Fusion  Peptides  and  their
      Interaction with Lipid Bilayers.**  *Biophys.  J.* 88 25-36 (2004).

27. Cavallo  L,  Kleinjung  J  and  Fraternali  F.   **POPS:  a  fast  algorithm  for  Solvent  Ac-
      cessible  Surface  Areas  at  atomic  and  residue  level.**   *Nuc.  Acids  Res.*  31  3364-3366 (2003).

28. George RA, Kleinjung J and Heringa J.  **Predicting protein structural domains from
      sequence  data.**  In  *Bioinformatics  and  Genomes  -  Current  Perspectives*,  chapter  1,
      1-26. Horizon Scientific Press (2003).

29. Kleinjung J, Fraternali F, Martin SR and Bayley PM. **Thermal Unfolding Simulations
      of Apo-calmodulin using Leap-dynamics.**  *Proteins*  50 648-656 (2003).

30. Lin  KX,  Kleinjung  J,  Taylor  WR  and  Heringa  J.   **Testing  homolgy  with  CAO:  A
      contact-based  Markov  model  of  protein  evolution.**   *Comp.  Biol.  Chem.*  27  93-102 (2003).

31. Simossis V, Kleinjung J and Heringa J. **An overview of Multiple Sequence Alignment.**
      In **Current Protocols in Bioinformatics*, chapter 3.7, 3.7.1-3.7.25. John Wiley & Sons (2003).

32. Kleinjung  J,  Douglas  N  and  Heringa  J.  **Parallelized  multiple  alignment.**
      *Bioinformatics*  18 1270-1271 (2002).

33. Kleinjung J, Bayley P and Fraternali F. **Leap-dynamics:  efficient sampling of conformational
      space  of  proteins  and  peptides  in  solution.**  *FEBS  Letters*  470(3)  257-262 (2000).

34. Kleinjung J and Fabry M.  **Photoreactive Insulin Derivatives for the Detection of the
      Doubly Labeled Insulin Receptor.**  *Peptides*  21 401-406 (2000).

35. Kleinjung J, Petit MC, Orlewski P, Mamalaki C A Liolitsas, Tzartos SJ, Tsikaris M
      V Sakarellos-Daitsiotis, Sakarellos C, Marraud M and Cung MT.  **The 3D structure
      of  the  Immunogenic  Region  of  the  Acetylcholine  Receptor:  A  Combined  2D-NMR,
      Homology and Molecular Modeling Approach.**  *Biopolymers*  53 113-128 (2000).

36. Martin SR, Lu AQ, Xiao J, Kleinjung J, Beckingham K and Bayley PM.  **Conformational and
      metal-binding properties of androcam, a testis-specific, calmodulin-related
      protein from Drosophila.**  *Protein  Sci.* 8 2444-2454 (1999).

37. King's   College   London   and   Deutsches   Wollforschungsinstitut.     **Hepatoselective
      Insulin-Thyroxin  Conjugates**,  *Patent  WO9965941*  (1999-12-23).

38. Deutsches  Wollforschungsinstitut.    **Phosphorylated  Insulins**,  *Patent  DE19535701*
     (A1)  (1997-03-27).

