---
layout: default
title: About Jens Kleinjung
---
<div class="blurb">
	<h1>Jens Kleinjung</h1>
	<p>About me. <a href="/about">More to come...</a></p>

Research=
* gene expression patterns and regulation
* protein sequence/structure/function analysis and design
* molecular dynamics and solvation
* interaction networks and information processing
We use computational methods to understand functional properties of biomolecules. 

The objective of our studies is to describe cellular processes at the molecular level
and to predict properties of biomolecular systems, in particular those associated with diseases.

[[Image:M32K25_anim.gif|500px|M32K25 structural alphabet]]
[[Image:M32K25_expel.gif|500px|protein fit using M32K25]]

=== Protein Function and Design ===
* [[Allostery]]
* [[Structural Alphabet]]
* [[Ras Functional Motions]]
* [[AFM Imaging]]
* [[Protein complexes]]
=== Sequence and Structure Alignment ===
* [[Homology-based Sequence Alignment]] : improved multiple alignment by homology extension
* [[ALICAO]] : contact-based sequence alignment
* [[CAO]] : contact mutation matrices
* [[Parallelized Multiple Alignment]] : fast sequence alignment through parallelisation
=== Molecular Dynamics and Solvation ===
* [[Solvent Forces]] : Implicit Solvent Forces in Molecular Dynamics simulations (with Walter R.P. Scott, Jane R. Allison, Wilfred F. van Gunsteren and Franca Fraternali)
* [[POPS* and POPScomp]] : solvent accessible surface area of proteins and nucleic acids
* [[Leap dynamics]] : combined Molecular Dynamics and Monte-Carlo simulations
=== Interaction Networks ===
* [[NetFeat]] : Analysis of protein-protein interaction networks (with F. Fraternali, L. Fernandes, A.C.C. Coolen and A. Annibale)

== Methods and Resources ==
* Sampling/Searching/Optimisation: Molecular Dynamics, Monte Carlo simulations, Genetic Algorithms
* Alignment/Modelling/Design: Dynamic Programming, Hidden Markov Models, Bayesian Models
* Combinatorics: Structural Alphabets, Suffix Trees

The computational resources in our lab comprise a High Performance Cluster with 1440 CPU cores
and 3 TB RAM.
The cluster runs on the [http://www.qlustar.com/ Qlustar Precise] software, data I/O is provided by a
25 TB [http://wiki.lustre.org/ Lustre] file system and the network backbone consists of 3
[http://en.wikipedia.org/wiki/Link_aggregation bonded Ethernet] interfaces per node.

== References ==
* [[Publications Jens Kleinjung]]
* [[CV Jens Kleinjung]]
* [[Lectures Jens Kleinjung]]

== Programs ==
Software packages can be found on my site [https://github.com/jkleinj GitHub jkleinj]
and on our [[Software]] page with additional links to servers.

Collections of one-liners:
* [[LaTeX]]
* [[UNIX]]

</div><!-- /.blurb -->

